config/deploy.rb

```diff
@@ -1,4 +1,4 @@
-lock '3.5.0'
+lock '3.14.1'
```

Comment:
Reminder here about the option to use `~>` (and maybe going ahead and committing the Capistrano version update you made locally too).

Reply:
Both good ideas - I'll add them to the other branch!